src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -374,6 +376,20 @@ impl CloudHypervisorInner {
             cmd.args(["--seccomp", "false"]);
         }
 
+        let netns = self.netns.clone();
+
+        unsafe {
+            let _pre = cmd.pre_exec(move || {
+                if let Some(netns_path) = &netns {
+                    info!(sl!(), "set netns for vmm master {:?}", &netns_path);
```

Comment:
Also, since that block runs in the child, the log message doesn't get logged, so could you move the log call to the main thread outside the `unsafe` block?